.babelrc

```diff
@@ -5,7 +5,7 @@
       {
         "useBuiltIns": true,
         "targets": {
-          "node": "4.3"
+          "node": "4.8"
```

Comment:
`>= 4` (if `babel-preset-env` (browserslist) supports it) ?

Reply:
Something to keep in mind here. I still think the minimum needs to be a static value as opposed to >= 4 ( for both preset-env & all the `engines` objects in the pacakge.json ).

Thinking being you will have an ever increasing minimum node version which would have a less than optimal UX.

It's also probably worth considering finding the lowest viable version number to set as the minimum. i.e. `4.6` as opposed to the latest NodeJS minor version. I'm sure there are plenty of people that aren't on 4.8 yet ( for reasons I will never comprehend )

Reply:
According to [the comment](https://github.com/npm/npm/issues/18395#issuecomment-328378921) on that thread by the npm staff, the earliest supported npm node version is `4.7`.

Would webpack packages using this change technically require a major version bump if we're no longer testing / supporting older node 4.x versions?

Reply:
I’m afraid people who aren’t on 4.8 aren’t on 4.6 either ;-| But I don’t know how to prove either of these points.

Reply:
Given that Node 8 is about to go LTS, I don't know if I feel terribly bad about pushing up the minimum 4.x version.

To @mattlewis92's point, following the strictest letter of the lay, yes it should be a major as it is breaking based on what is currently supported.

Give Webpack follows the LTS schedule of NodeJS & v8 goes LTS this in October, the least disruptive path for the community may be to just drop 4.x when v8 officially takes over as the latest LTS.

In the mean time, the Travis config can be modified to accomodate the issue with 4.3 & npm@5

Reply:
@d3viant0ne just change only travis configuration?

Reply:
The `Buffer` related backports landed in `v4.5.0`, no idea about the if and when V8 semver minor update(s) happend in the ` node v4+` line. Normally within a major range it is mandatory to update to the latest `minor/patch` whenever possible (security etc pp). If for any corporate reasons this isn't possible, imho at some point it's just their problem and getting webpack to run is one of the smaller ones they have then 😛